src/package/package_api.js

```diff
@@ -84,7 +84,7 @@ function defineReadOnlyProperty(object, key, value) {
 function PackageInformation(json) {
   for (var field in json) {
     var val = json[field];
-    if (field === 'installDate')
+    if (field === 'lastModified')
```

Comment:
The 2.3.0 spec(https://developer.tizen.org/dev-guide/2.3.0/org.tizen.mobile.web.device.apireference/tizen/package.html#PackageInformation) interface PackageInformation is same as 2.2.1.
However, PackageManager interface is different, hope fix with BUG=XWALK-3153